### PR TITLE
sort projects by marketcap

### PR DESCRIPTION
**Pull request by GarageInc**
**Summary**

Search sorted by marketcap volume

**Screenshots**
BEFORE
image

AFTER
image

### CODE REVIEW

**Comment by Partysun** [comment]
@GarageInc is Marketcap always accessible here?

File: src/components/Search/SearchContainer.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ export const SearchContainer = ({
       className={cx(styles.wrapper, className)}
       iconPosition='left'
       onSuggestionSelect={({ slug }) => history.push(`/projects/${slug}`)}
+      sorter={({ marketcapUsd: a }, { marketcapUsd: b }) => b - a}
```
Comment (haritonasty) [inline src/components/Search/SearchContainer.js]: let's use `rank` instead `marketcapUsd` (we will have that field in our json, as I understand)

Comment (GarageInc) [reply src/components/Search/SearchContainer.js]: @Partysun , @haritonasty here are using such query
image
I will add a rank for it

Comment (GarageInc) [reply src/components/Search/SearchContainer.js]: done

**Pull request merged**